packages/node_modules/samples/browser-plugin-meetings/index.html

```diff
@@ -182,6 +198,10 @@ <h2>Meeting Controls</h2>
             <button id="gc-dtmf-tones-send" type="button" onclick="sendDtmfTones()">meeting.sendDTMF()</button>
             <label id="gc-dtmf-status"></label>
           </div>
+          <div class="transcription">
+            <button id="gc-start-receiving-transcription" disabled type="button" onclick="startReceivingTranscription()">start receiving transcription (click me before joining)</button>
+            <textarea id="gc-transcription-content" disabled>Transcription Content: Webex Assistant must be enabled, check the console!</textarea>
```

Comment:
The idea of this being a text area was that the transcripts would show on the app itself. if you're differing to just showing it in the console, that's fine but then there's no need for this text area

Reply:
im not logging out anymore. just the textarea for now.